test/test_session.py

```diff
@@ -245,10 +245,12 @@ def test_pool_lifo(self):
         b.end_session()
 
         s = self.client.start_session()
+        self.client.admin.command("ping", session=s)
```

Comment:
Can we remove these pings now that `s.session_id` materializes the session?

Reply:
No, this test requires the sessions actually enter the pool, which materializing itself does not do. The test fails without these pings, sadly.

Reply:
I mean replace this ping with `s.session_id`.

Reply:
Can you explain? It appears these tests are only checking `session_id`.

Reply:
The test checks that session `a` and `b` are both in the session pool after they are ended, with `b` at the top due to LIFO ordering. We expect the next session created to be `b`, but since session creation no longer interacts with the pool unless a timeout is present, materializing the new session instead of using a `ping` results in a new session entirely.

Reply:
Oh that's a bug in get_server_session. After it's fixed we should remove the pings here.